artiq/tools.py

```diff
@@ -155,12 +180,22 @@ def multiline_log_config(level):
     root_logger.addHandler(handler)
 
 
-def init_logger(args):
-    multiline_log_config(
-        level=logging.WARNING + args.quiet*10 - args.verbose*10)
+def init_logger(args: argparse.Namespace) -> NOne:
```

Comment:
Typo: NOne (doesn't the interpreter throw an error when you run this file? Type annotations are still evaluated)
Anyway I think type annotations should not be used.